mage/tests/kind.config.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "kind.x-k8s.io/v1alpha4"
```

Comment:
Same comment as above, these test config files aren't needed if you use the targets in the magefiles repo

Reply:
Removed